Source/WebKit/GPUProcess/graphics/wc/RemoteWCLayerTreeHost.cpp

```diff
@@ -44,14 +44,16 @@ std::unique_ptr<RemoteWCLayerTreeHost> RemoteWCLayerTreeHost::create(GPUConnecti
 }
 
 RemoteWCLayerTreeHost::RemoteWCLayerTreeHost(GPUConnectionToWebProcess& connectionToWebProcess, WebKit::WCLayerTreeHostIdentifier identifier, uint64_t nativeWindow, bool usesOffscreenRendering)
-    : m_connectionToWebProcess(connectionToWebProcess)
+    : m_workQueue(IPC::StreamConnectionWorkQueue::create("RemoteWCLayerTreeHost work queue"))
```

Comment:
ANGLE D3D renderer isn't thread-safe. WebGL and TextureMaper should be run in the same thread of GPU process. Don't create a new work queue.